### PR TITLE
修复custom_logo.enable字段无用的BUG；给Gemini和Pisces添加custom_logo支持

### PR DESCRIPTION
**Pull request by chenghm123**
1、修复custom_logo.enable字段无用的BUG
2、给Gemini和Pisces添加custom_logo支持

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -136,10 +136,10 @@ menu_settings:
 # ---------------------------------------------------------------
 
 # Schemes
-scheme: Muse
```
[the remark this answers is not in the document]
Comment (chenghm123) [reply _config.yml]: Repair "custom_logo.enable" doesn't work .
Adding custom_logo support of schema Gemini and Pisces .

[the remark this answers is not in the document]
Comment (chenghm123) [reply _config.yml]: Repair "custom_logo.enable" doesn't work. Adding custom_logo support of schema Gemini and Pisces.

**Comment by chenghm123** [comment]
Sorry! I'm first pull.

**Comment by sli1989** [comment]
Can u re-modify the pr according to the new [file](https://github.com/theme-next/hexo-theme-next/blob/3b2f980210924d2df4d7d80b0ed71e34332ea3fc/layout/_partials/header/brand.swig) where `config.title` to `title`, and the added section maybe added before the line12.

https://github.com/theme-next/hexo-theme-next/blob/3b2f980210924d2df4d7d80b0ed71e34332ea3fc/layout/_layout.swig#L7

**Comment by stevenjoezhang** [comment]
Superseded by #520

**Pull request closed without merging**